scripts/pythran

```diff
@@ -149,6 +150,9 @@ try:
 
     module_name, ext = os.path.splitext(os.path.basename(args.input_file))
 
+    # Add compiled module path to search for imported modules
+    sys.path.append(os.path.dirname(args.input_file))
+
```

Comment:
sorry to comment now that travis is ok, but ``scripts/pythran`` seems a bad place to add this: what if we use pythran as a library ? the path would not be added... (as in ``%%pythran`` magic)